filesystem/File.php

```diff
@@ -390,7 +390,6 @@ public function getCMSFields() {
 
 		//get a tree listing with only folder, no files
 		$folderTree = new TreeDropdownField("ParentID", _t('AssetTableField.FOLDER','Folder'), 'Folder');
-		$folderTree->setChildrenMethod('ChildFolders');
```

Comment:
Can you explain why we don't use this anymore? I thought the fix was to only count folders not all children?